internal/provider/kubernetes/controller.go

```diff
@@ -1651,6 +1652,9 @@ func (r *gatewayAPIReconciler) processEnvoyProxy(ep *egv1a1.EnvoyProxy, resource
 	if err := validation.ValidateEnvoyProxy(ep); err != nil {
 		return fmt.Errorf("invalid envoyproxy: %w", err)
 	}
+	if err := bootstrap.Validate(ep.Spec.Bootstrap); err != nil {
```

Comment:
Can we cover this validation with a positive/negative test?

Reply:
That's already included here: https://github.com/envoyproxy/gateway/pull/4116/files/c790f2e378c00d169e271049d17c050fb488dd86#diff-6783c0822a78972a488213dd33bef82297eaac567e74fd5ae639020e18448e3c